version/version.go

```diff
@@ -14,6 +14,14 @@ var (
 	vcsCommit string
 )
 
+var (
+
+	// DefaultTag is the default repo for images
+	DefaultRepo = "ghcr.io/azure"
+	// DefaultTag is the default tag for images
+	DefaultTag = "latest"
```

Comment:
should we use https://github.com/Azure/eraser/blob/main/version/version.go#L10?

Reply:
`buildversion` has the commit hash attached, example: `-X 'github.com/Azure/eraser/version.buildVersion=v1.0.0-beta.3.24-049b7d2c902a3d'`

Reply:
if you are on a git tag, it shouldn't show the commit. 

for example:
```
$ ./build/version.sh  
-X 'github.com/Azure/eraser/version.buildTime=1675117790' -X 'github.com/Azure/eraser/version.vcsCommit=34736043ef5cef29774745cd8e95029dde48045d' -X 'github.com/Azure/eraser/version.vcsState=clean' -X 'github.com/Azure/eraser/version.buildVersion=v1.0.0-rc.1' -X 'github.com/Azure/eraser/version.vcsMajor=1' -X 'github.com/Azure/eraser/version.vcsMinor=0'
```

Reply:
But then during development, deploying the helm chart in `manifest_staging/charts/eraser` will default to using `v.1.0.0-beta.2.24-049b7d2c902a3d` while `make docker-build-eraser` will produce images with the tag `v1.0.0-beta.3`.

Then the developer needs to manually match up the tags every time they deploy the helm chart to test it, which is kinda surprising behavior.

Another other thing we could do is provide a default tags in `values.yaml` that match VERSION in the Makefile, but we'd need to update it every time there's a new release. We could just change the release script to update them when a new version releases.